src/components/timeline/TrackIPC.js

```diff
@@ -15,7 +15,7 @@ import type { ThreadIndex, Milliseconds } from 'firefox-profiler/types';
 
 import type { ConnectedProps } from 'firefox-profiler/utils/connect';
 
-import './TrackIPC.css';
+import 'firefox-profiler/components/timeline/TrackIPC.css';
```

Comment:
Please revert this line, it starts with `./` and it's not related to `src/actions`.